demo/meeting/src/utils/VersionLabel.tsx

```diff
@@ -3,9 +3,11 @@
 
 import React from 'react';
 import { Versioning } from 'amazon-chime-sdk-component-library-react';
+import { Versioning as SDKVersioning } from 'amazon-chime-sdk-js';
 
 export const VersionLabel = () => {
   const versionTag = `${Versioning.sdkName}@${Versioning.sdkVersion}`;
+  const sdkVersionTag = `amazon-chime-sdk-js@${SDKVersioning.sdkVersion}`;
```

Comment:
can you please change to pull the name using ```SDKVersioning.sdkName()```

Reply:
Fixed